src/main/java/org/apache/skywalking/banyandb/v1/client/BanyanDBClient.java

```diff
@@ -411,6 +439,26 @@ private void defineIndexRules(Measure measure, List<IndexRule> indexRules) throw
         irbRegistry.create(binding);
     }
 
+    /**
+     * Try to find the group defined
+     *
+     * @param name name of the group
+     * @return the group found in BanyanDB
+     */
+    public Group findGroup(String name) throws BanyanDBException {
```

Comment:
```suggestion
    public Optional<Group> findGroup(String name) throws BanyanDBException {
```
Consider using Optional if this is expected to nullable. We are using JDK8+, right?

Reply:
Yes. We could do this.